rfcs/convergence/first-rule-of-aria.md

```diff
@@ -0,0 +1,164 @@
+<button class="button-class">This is a Simple Button</button>
+<!-- Aria version -->
+<div class="button-class" tabindex="0" role="button">This is a div that looks and behaves as a Button</div>
+<!-- Style only -->
+<div class="button-class">
+  This is a div that looks like Button but doesn't behave as such therefore, I can add a
+  <a href="/somewhere">Link inside of it</a>
+</div>
+```
+
+---
+
+If isn't equivalent to native elements but can benefit from using native elements in it's implementation - `Accordion`, `Carousel`, `Disclosure`, `MenuButton`, `Menu`, `MenuBar`, `Tabs`
+
+- Should treat internal elements that could follow previous category as such (e.g: `Accordion` has an internal element with role `button`, this element should previous category)
+- Provide authors an option to use a semantic HTML version (e.g. Dropdown should have an author option to use a `<select>` element, even if we also allow a custom version with a stylable extendable options list).
+
+##### Accordion Example
+
+```tsx
+<Accordion>
+  <AccordionItem>
+    <AccordionHeader>Header</AccordionHeader>
+    <AccordionPanel>Content</AccordionPanel>
+  </AccordionItem>
+</Accordion>
+
+<Accordion>
+  <AccordionItem as="div">
+    <AccordionHeader button={{as: "div", role: undefined, tabIndex: undefined}}>
+      Header <Link to="/somewhere">Link</Link>
+    </AccordionHeader>
+    <AccordionPanel>Content</AccordionPanel>
+  </AccordionItem>
+</Accordion>
+```
+
+```html
+<!-- accordion header -->
+<h3 class="header-class">
+  <button id="header-1" aria-disabled="false" class="header-button-class" aria-controls="panel-1">
+    Header
+  </button>
+</h3>
+<!-- accordion panel -->
+<div id="panel-1" role="region" aria-labelledby="header-1" class="panel-class">
+  Content
+</div>
+
+<!-- accordion header -->
+<div role="heading" class="header-class">
+  <div id="header-1" aria-disabled="false" class="header-button-class" aria-controls="panel-1">
+    Header <a href="/somewhere">Link</a>
+  </div>
+</div>
+<!-- accordion panel -->
+<div id="panel-1" role="region" aria-labelledby="header-1" class="panel-class">
+  Content
+</div>
+```
+
+---
+
+If isn't equivalent to native elements and can detriment from using native elements in it's implementation - `Table`, `List`, `Tree View`, `Tree Grid`, `Grids`
+
+- Should not follow 1st rule of ARIA
+- In the case of components that simply cannot follow this pattern a session in the Spec of that component
+  should be dedicated to explain why this pattern hasn't being followed, to avoid future attempts of converting the component for the pattern.
+
+---
+
+### Pros and Cons
+
+#### Pros
+
+- Better support overall for A11y
+
+#### Cons
+
+- Styles override to remove styles from semantic elements will be necessary
+- In cases of heading, a lvl attribute might be required
+- In cases of virtualization, semantic elements are not recommended (big problem for lists, menus, tables, and other components that depend on a group of items)
```

Comment:
might be worth specifying this is because of performance costs of layout recalc, especially for table (does anyone know if this is also true for lists?)